packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -113,6 +114,27 @@ export function authorizationServiceBuilder(
         showUsers: client.consumerId === organizationId,
       };
     },
+    async getClients(
+      filters: GetClientsFilters,
+      { offset, limit }: { offset: number; limit: number },
+      authData: AuthData,
+      logger: Logger
+    ): Promise<ListResult<Client>> {
+      logger.info(
+        `Retrieving clients by name ${filters.name} , userIds ${filters.userIds}`
+      );
+      const userIds = authData.userRoles.includes("security")
+        ? [authData.userId]
+        : filters.userIds.map(unsafeBrandId<UserId>);
```

Comment:
```suggestion
        : filters.userIds;
```
They are already `UserId`, no need to convert them again